### PR TITLE
Update deps commits of opencensus to support building with bzl 0.25.x

### LINKED ISSUE #4810: ray installation broke with bazel 0.25.x
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: 19.04
- **Ray installed from (source or binary)**: source
- **Ray version**: latest
- **Python version**: 3.6 
- **Exact command to reproduce**: `pip install -e . --verbose`

### Describe the problem

pip install would fail with bazel version 0.25.x but bazel 0.24.1 works just fine. This is likely caused by https://github.com/census-instrumentation/opencensus-cpp/pull/307

### PR DESCRIPTION
**Pull request by jovany-wang**
## What do these changes do?
With bazel 0.25.x released, some users build Ray from source with bazel 0.25.x.
We should update `opencensus` and `absl` to support building with bzl 0.25.x.

I have tested this PR with bazel 0.21.x, 0.22.x and 0.25.x.

## Related issue number
This closes #4810 and #4853

 `opencensus` fixed the issue in https://github.com/census-instrumentation/opencensus-cpp/pull/316

### CODE REVIEW

**Comment by jovany-wang** [comment]
@tmorgan4  Could you please try to build Ray with this PR with bazel 0.25.x? I think this can fix your issue.

File: bazel/ray_deps_setup.bzl [hunk 1]
```diff
@@ -80,18 +80,18 @@ def ray_deps_setup():
...
+        strip_prefix = "opencensus-cpp-3aa11f20dd610cb8d2f7c62e58d1e69196aadf11",
+        urls = ["https://github.com/census-instrumentation/opencensus-cpp/archive/3aa11f20dd610cb8d2f7c62e58d1e69196aadf11.zip"],
     )
    
     # OpenCensus depends on Abseil so we have to explicitly pull it in.
     # This is how diamond dependencies are prevented.
     git_repository(
         name = "com_google_absl",
-        commit = "88a152ae747c3c42dc9167d46c590929b048d436",
+        commit = "5b65c4af5107176555b23a638e5947686410ac1f",
```
Comment (robertnishihara) [inline bazel/ray_deps_setup.bzl]: This commit is from March 27, right? 

https://github.com/abseil/abseil-cpp/commit/5b65c4af5107176555b23a638e5947686410ac1f

Why not a more recent one? Just curious.

**Review by pcmoritz — APPROVED** [review]
LGTM

File: bazel/ray_deps_setup.bzl (hunk 1 above)
Comment (jovany-wang) [reply bazel/ray_deps_setup.bzl]: There is no special requirements to use this commit. Just it was mentioned in https://github.com/ray-project/ray/issues/4810.  The most recent one on the master branch of absl should be ok as well.

**Pull request merged**